### DISCUSSION TITLE
Preserve 'Variable' kind entries by Updating Classpaths Wizard?

### DISCUSSION THREAD

**Discussion by haubi** [eclipse-pde/eclipse.pde#532] [Ideas]
"Update the classpath settings" button (Manifest Editor Overview) or "Update Java Classpath" dialog (Plug-in Tools->Update Classpath...) unconditionally _removes_ classpath entries of kind="var" (variable), as discovered along PR #497.

Although we don't (seem to) use 'variable' classpath entries in our application here, unconditionally removing feels counter-intuitive.

Is there a specific reason not to preserve them just like the other classpath entries?

**Comment by iloveeclipse** [comment]
I assume this is a simple bug.

**Reply by haubi** [reply]
Then I'm going to fix it along PR #497.
